components/automate-gateway/api/nodes/nodes.proto

```diff
@@ -42,6 +55,14 @@ service NodesService {
 			action: "infra:nodes:get"
 		};
 	};
+
+	/*
+	Update a node
+
+	PUT operation to update the details for a node, such as the name, fqdn, tags, or associated credentials.
+	Please note that this is a PUT operation, so all node details included in the create function
```

Comment:
Does this mean "This PUT operation overwrites ALL node details and requires the complete set of node details, consisting of a FQDN or IP address, a user-specified name, and a ssh or winrm credential reference, 
Substitute the desired values for the existing node details in the PUT message.

Reply:
yes!!! awesome, thank you, ill update it to say that